src/vs/code/electron-main/app.ts

```diff
@@ -531,6 +522,10 @@ export class CodeApplication extends Disposable {
 		const encryptionChannel = createChannelReceiver(encryptionMainService);
 		electronIpcServer.registerChannel('encryption', encryptionChannel);
 
+		const mainKeyboardLayoutService = accessor.get(IMainKeyboardLayoutService);
+		const mainKeyboardLayoutChannel = createChannelReceiver(mainKeyboardLayoutService);
+		electronIpcServer.registerChannel('mainKeyboardLayout', mainKeyboardLayoutChannel);
```

Comment:
Again just a small detail, typically our channel names don't reference the main side, so I suggest `keyboardLayout` as channel name.

Reply:
:+1: